nfs-ganesha/build/build_deb

```diff
@@ -27,6 +34,7 @@ setup_pbuilder
 ## Build the source tarball
 
 cd $WORKSPACE/nfs-ganesha
+PACKAGE_MANAGER_VERSION="`git describe --long | 'sed s/V//1'`-1xenial"
```

Comment:
Hardcoding ``xenial`` won't work if this job is used to build any other version. I think you can get the value you want here from the ``$DIST`` variable.  We should probably change that for every instance of ``xenial``.